### PR TITLE
Feature - trigger functions

### PR DESCRIPTION
**Pull request by mattiarossi**
Refs #38 ,
- implements trigger functions
- added a test for checking correct generation
- re-used decoration syntax used for volatility (//@plv8ify-trigger)

### CODE REVIEW

File: README.md
Comment (divyenduz) [inline README.md:39]: Is it a convention to call it `testRow`? How about just `Row` or `TestRow` (matching casing for types) or `RowType`?

**Review by divyenduz — APPROVED** [review]
LGTM, just added a nit-picky comment about the type name (not a blocker), will wait for your response on that before merging! Thanks for the contribution.

File: README.md
Comment (mattiarossi) [reply README.md:39]: > Is it a convention to call it `testRow`?

no, not at all, naming of the type is independent as it will only be used for type-checking in typescript. WHat matters is that the fields in the type match the table record names and types so that the trigger does not error out when run in plv8.
will change it to Row

**Comment by divyenduz** [comment]
Thanks for the contribution, this is live in `0.0.53`

**Pull request merged**
